package.json

```diff
@@ -43,6 +43,7 @@
     "eyes.it": "latest",
     "gh-pages-auto-release": "latest",
     "haste-preset-yoshi": "latest",
+    "identity-obj-proxy": "^3.0.0",
```

Comment:
why do you need it? the `stylable-proxy` should take care of it as well

Reply:
Withot it the icons are not working, can't import them